simpleclient_dropwizard/src/main/java/io/prometheus/client/dropwizard/DropwizardExports.java

```diff
@@ -20,15 +20,29 @@
 public class DropwizardExports extends io.prometheus.client.Collector implements io.prometheus.client.Collector.Describable {
     private static final Logger LOGGER = Logger.getLogger(DropwizardExports.class.getName());
     private MetricRegistry registry;
+    private MetricFilter metricFilter;
```

Comment:
Hmm, where is this defined?

Please import this explicitly, rather than relying on wildcards.

Reply:
This file has already had `import com.codahale.metrics.*;`, do you want me to split this wildcard into concrete classes?

Reply:
Yes please

Reply:
Done